webbpsf/webbpsf_core.py

```diff
@@ -856,7 +899,7 @@ def _get_fits_header(self, result, options):
                                           "[arcmin] Det. pos. in telescope V2,V3 coord sys"), after=True)
         result[0].header.insert("DET_V2", ('DET_V3', v2v3pos[1].value,
                                            "[arcmin] Det. pos. in telescope V2,V3 coord sys"), after=True)
-        result[0].header["APERNAME"] = (self._detectors[self._detector], "SIAF aperture name")
+        hdulist[0].header["APERNAME"] = (self._aperturename, "SIAF aperture name")
```

Comment:
I updated this as part of the generalization, because the GitHub review wouldn't let me leave a comment here. I simply copy/pasted from the NIRCam class, but forgot to change the variable name to `result` rather than  `hdulist`.

Reply:
Nevermind about the change request. I just went ahead and fixed my own typo.